testcafe/service.js

```diff
@@ -0,0 +1,22 @@
+import ServicePage from './pages/ServicePage';
+
+const servicePage = new ServicePage();
+
+fixture `Service Page`
+  .page(servicePage.url(1));
+
+test('Confirm Page Loads with Information', async (t) => {
+  const serviceName = 'Women\'s clinicA service offered by New Resource Name';
```

Comment:
This works on local, but not the testing suite. Going to switch to check for existence.